evaluation.html

```diff
@@ -1,68 +1,70 @@
 <DOCTYPE html>
     <html>
-        <head><title></title>
 
-            <link rel="stylesheet" type="text/css" href="css/evaluation.css">
-        </head>
+    <head>
+        <title></title>
 
-        <body>
+        <link rel="stylesheet/css" type="text/css" href="assets/css/normalize.css" />
+        <link rel="stylesheet/less" type="text/css" href="assets/css/evaluation.less">
+        <script src="less.js" type="text/javascript"></script>
+    </head>
 
-            <header>
-                <a id="homeLogo" href="login.html"><img src="assets/Logo.png"></a>   
-                <a class="pgSwitch borderSwitch" href="evaluation.html">Evaluations</a>
-<!--                @Alex there are no restrictions on number of classes-->
-                <a class="pgSwitch" href="New%20evaluation.html">New Evaluation</a>
-                <button id="Logout" type="button", Value="Logout", name="Logout">Logout</button>
+    <body>
 
+        <header class="header">
+            <a class="homeLogo" href="login.html"><img src="assets/images/Logo.png" class="header-img"></a>
+            <a class="pgSwitch borderSwitch" href="evaluation.html">Evaluations</a>
+            <a class="pgSwitch" href="NewEvaluation.html">New Evaluation</a>
+            <button class="Logout" type="button" , Value="Logout" , name="Logout">Logout</button>
```

Comment:
I know for a fact that I've seen these stray commas before. When are you planning on cleaning them up? :)